src/TargetingPack/Microsoft.Internal.Extensions.Refs/ref/Microsoft.Internal.Extensions.Refs.csproj

```diff
@@ -32,11 +32,16 @@
 
     <!-- Reference implementation assemblies in addition to ref assemblies to get xml docs -->
     <ReferenceImplementationAssemblies>true</ReferenceImplementationAssemblies>
+    <!-- We are ignoring MSB3243 warnings since implemenation and reference assemblies are versioned differently. We need both to compose the targeting pack with reference assemblies and xml docs. -->
+    <MSBuildWarningsAsMessages>MSB3243</MSBuildWarningsAsMessages >
+
+    <LayoutTargetDir>$(ArtifactsObjDir)TargetingPack.Layout\$(Configuration)\</LayoutTargetDir>
   </PropertyGroup>
 
   <ItemGroup>
     <!-- Don't reference anything unless building a targeting pack. Otherwise will use ref and non-ref assemblies. -->
-    <Reference Include="@(ProjectReferenceProvider)" Condition="$(IsPackable)" />
+    <Reference Include="@(ProjectReferenceProvider)" Condition="$(IsPackable)" ReferenceOutputAssembly="false" />
+    <Reference Remove="Microsoft.Extensions.Hosting.Systemd" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
Why do we need to remove this 1 Ref?

Reply:
This package is special since it targets netstandard2.1 only. I don't think any of our other projects uses this dependency though.